src/webhook/handlers/groupMessage.js

```diff
@@ -49,6 +51,11 @@ export default async function processText(event, groupId) {
     return { event, groupId, replies };
   }
 
+  if (event.input.toLowerCase() === LEAVE_KEYWORD) {
+    await lineClient.post(`/${event.source.type}/${groupId}/leave`);
```

Comment:
Does this trigger leave webhook? If not, we may need to manually send `ga` here.

Reply:
Yes, it will.